packages/generate-persisted-query-manifest/src/__tests__/cli.test.ts

```diff
@@ -925,6 +926,71 @@ export default config;
   await cleanup();
 });
 
+test("can specify custom document transform in config", async () => {
```

Comment:
Should probably add a test for Apollo Client 3.7 since document transforms aren't run. Will do this after we figure out the right approach.

Reply:
Are you going to do this? I'll approve but feel free to follow through on this idea or not.

Reply:
Ah yes thanks! I'll add this before merging. Appreciate the approval!

Reply:
Turns out this is a bit tougher than I originally thought because its difficult to control module resolution on the CLI package itself in tests. While I can install Apollo Client 3.7 which is properly used in the config itself, the cli still resolves to the latest version of the client, so the transform is still run.

I'll test locally with a built package against a repo with 3.7 just to make sure I didn't miss anything, but I feel pretty confident this is ok otherwise.

Reply:
Added an update to provide a nicer type when using Apollo Client 3.7 where `DocumentTransform` is not available: https://github.com/apollographql/apollo-utils/pull/412/commits/d7475fa6a1b5b68aeb8ffcbbce4ad9fc14dad858. Now when trying to use that option when that version is installed, the only valid value is `undefined`.